CONTRIBUTING.rst

```diff
@@ -0,0 +1,9 @@
+Contributing to StackStorm
```

Comment:
I've added this file because the link to it is displayed when creating a PR - https://github.com/blog/1184-contributing-guidelines